### DISCUSSION TITLE
How to make subsequent login prompt for password like first time?

### DISCUSSION THREAD

**Discussion by TLKG** [abacritt/angularx-social-login#402] [Q&A]
Playing the demo, first time log in works out great! Click on sign out button, redirected back to initial screen of choices. Click on G again, it took me right in without asking for password. How to make this log in after sign-out prompt for password? In other words, after sign-out, re-sign in is expected to go through the same email/password input again.

**Comment by jaibatrik** [comment]
Check if calling `signOut(true)` helps.

**Reply by SeredaOM** [reply]
This worked for me, thanks!
